brokenspoke_analyzer/core/compute.py

```diff
@@ -22,10 +24,12 @@ def execute_sqlfile_with_substitutions(
     engine: Engine,
     sqlfile: pathlib.Path,
     bind_params: typing.Optional[typing.Mapping[str, typing.Any]] = None,
+    log_percentage: typing.Optional[float] = None,
 ) -> None:
     """Execute SQL statements with substitutions."""
-    logger.debug(f"Execute {sqlfile}")
-    logger.debug(f"{bind_params=}")
+    if log_percentage == None or uniform(0, 1) <= log_percentage:
```

Comment:
The logging is extremely noisy can calculating roads and census blocks individually. I wasn't sure on the best method to log _some_ during the longer calculation steps, but ended up with this as a configurable way to be able to track progress.

Reply:
Yeah, not sure about this though. I think I prefer to have too much logs rather than a sample. Maybe we could just increase the `log_level` to `trace` in this case?

Reply:
Yes, that makes sense!